heim-process/src/sys/windows/bindings/handle/limited_info.rs

```diff
@@ -139,4 +141,12 @@ impl ProcessHandle<QueryLimitedInformation> {
             Ok((creation, exit, kernel, user))
         }
     }
+
+    pub fn owner(&self) -> ProcessResult<User> {
+        // TODO clean this up?
+        match Token::open(&self.handle)?.user() {
+            Ok(x) => Ok(x),
+            Err(e) => Err(e.into()),
+        }
```

Comment:
That should work probably?

```suggestion
        pub fn owner(&self) -> ProcessResult<User> {
            Token::open(&self.handle)?.user().map_err(Into::into)
        }
```